poleno/utils/tests/test_templatetags.py

```diff
@@ -250,41 +226,92 @@ def test_active_filter(self):
         self.assertEqual(r3.content, u'(index=False)(first=False)(second=True)(second:first=False)')
         self.assertEqual(r4.content, u'(index=False)(first=False)(second=True)(second:first=True)')
 
+class ChangeLangTemplatetagTest(TestCase):
+    u"""
+    Tests ``change_lang`` template tag. Tests are performed by requesting view using template with
+    this tag.
+    """
+    def language_view(request, *args, **kwargs):
+        return HttpResponse(Template(
+            u'{% load change_lang from poleno.utils %}'
+            u'({% change_lang "en" %})'
+            u'({% change_lang "de" %})'
+            u'({% change_lang "fr" %})'
+        ).render(Context({
+            u'request': request,
+        })))
+
+    def page_not_found(request):
+        return HttpResponseNotFound(Template(
+            u'{% load change_lang from poleno.utils %}'
+            u'({% change_lang "en" %})'
+            u'({% change_lang "de" %})'
+            u'({% change_lang "fr" %})'
+        ).render(Context({
+            u'request': request,
+        })))
+
+    urlpatterns = tuple(i18n_patterns(u'',
+        url(r'^language/$', language_view, name=u'language'),
+        url(r'^language/(?P<a>\d+)/(?P<b>\d+)/$', language_view, name=u'language_kwargs'),
+        url(r'^language/(.+)/(.+)/(.+)/$', language_view, name=u'language_args'),
```

Comment:
Uz len detail, ze by som jednotlive testovaci url rozlisil nejakym textom v ramci URL. Nech potom v teste je zjavne, ktora URL v ktorom teste pouziva. Napriklad:

```suggestion
        url(r'^language/$', language_view, name=u'language'),
        url(r'^kwargs/(?P<a>\d+)/(?P<b>\d+)/$', language_view, name=u'language_kwargs'),
        url(r'^args/(.+)/(.+)/(.+)/$', language_view, name=u'language_args'),
```

Reply:
:ok: vytvoril som na to vlastny commit